Documentation/spec/docfx_document_schema.md

```diff
@@ -48,29 +48,13 @@ This is the root document object for *THIS schema*.
 
 | Field Name      | Type   | Description
 |-----------------|--------|----------
-| schemaVersion | string | `*`The version of the schema specification
-| fileType      | string | `*`The file type of the document model. Supported values are `YAML` and `JSON`.
-| type            | string | `*`The type of the root document model MUST be `object`
-| mimeType      | string | The mime type of the document model that applies to current schema. If not set, the schema name will be considered as its mime type.
-| info          | [Info Object](#info-object) | `*`The information about current schema
-| properties      | [Property Definitions Object](#property-definitions-object) | An object to hold the schema of all the properties if `type` for the model is `object`
-
-##### Patterned Field
-| Field Name | Type | Description
-|------------|------|----------
-| ^x-        | Any  | Allows extensions to *THIS schema*. The field name MUST begin with x-, for example, x-internal-id. The value can be null, a primitive, an array or an object.
-
-#### Info Object
-The object provides metadata about the schema.
-
-##### Fixed Field
-
-| Field Name      | Type   | Description
-|-----------------|--------|----------
-| version      | string | `*`The version of current schema object
-| id           | string | It is best practice to include an `id` property as an unique identifier for each schema.
-| title        | string | The title of current schema
-| description  | string | A short description of current schema
+| $schema         | string | `*`The version of the schema specification, for example, `https://github.com/dotnet/docfx/v1.0/schema#`.
+| version         | string | `*`The version of current schema object.
```

Comment:
version is for current schema object instead of the specification

---
In reply to: [126855679](https://github.com/dotnet/docfx/pull/1854#discussion_r126855679) [](ancestors = 126855679)